infra/gateway/src/main/resources/application-dev.yml

```
@@ -28,3 +28,8 @@ spring:
           uri: lb://user-service
           predicates:
             - Path=/api/users/**
+
+  redis:
+    data:
+      host: localhost
+      port: 6379
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)